guardian/prosemirror-elements
# chore: Add prosemirror-dev-tools as a dependency

## What does this change?

At the moment, we use a CDN to source the prosemirror-dev-tools. Why do that, when you can include it as a npm package? 

This PR also tidies up the `index.html` a bit, and removes an `index.html` in the root of the project that as far as I can tell is redundant.

## How to test

Run `yarn` in the project root, and then run the project normally. You should still see the devtools icon in the right hand side of the page. Open it – the devtools should work as expected: 

![tr-dev-tools](https://user-images.githubusercontent.com/7767575/188085787-365c9167-b83b-434b-b295-0fa63417d2fb.gif)
